src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -231,7 +234,11 @@ public Config validate(Map<String, String> connectorConfigs) {
     SnowflakeConnectionService testConnection;
     try {
       testConnection =
-          SnowflakeConnectionServiceFactory.builder().setProperties(connectorConfigs).build();
+          SnowflakeConnectionServiceFactory.builder()
+              .setProperties(connectorConfigs)
+              .setNetworkTimeout(VALIDATION_NETWORK_TIMEOUT_MS)
```

Comment:
Should we added it to `connectorConfigs` or move this to the same place whether you added the other timeout constants? What's so special about this one?

Reply:
Mentioned above. 
These timeouts is for checking if db exists, schema exists etc. (Connection through JDBC)